### PR TITLE
Do not fail feed update when unable to get one episode's file size

### LINKED ISSUE #57: Unable to get size of failed video
Hello,
If a video DL is failed due to #54 , podsync is unable to get its size and the feed update fails : 

```
podsync    | time="2019-11-25T18:10:45Z" level=error msg="youtube-dl error: [youtube] k5qMiJwbCo4: Downloading webpage\n[youtube] k5qMiJwbCo4: Downloading video info webpage\nERROR: This video contains content from FranceTV Distribution, who has blocked it on copyright grounds.\nSorry about that.\n" episode_id=k5qMiJwbCo4 error="failed to execute youtube-dl: exit status 1" index=35
podsync    | time="2019-11-25T18:10:45Z" level=error msg="failed to update feed: https://www.youtube.com/channel/UCNovJemYKcdKt7PDdptJZfQ/videos" error="failed to get episode file size: stat /app/data/jmj/k5qMiJwbCo4.mp3: no such file or directory"
```

### PR DESCRIPTION
**Pull request by b-abadie**
This seems to fix #57 
Beware ! These are my very first lines of go. Merge with caution. :grinning:

### CODE REVIEW

File: cmd/podsync/updater.go [hunk 1]
```diff
@@ -103,13 +110,6 @@ func (u *Updater) Update(ctx context.Context, feedConfig *config.Feed) error {
 			logger.Debug("skipping download of episode")
 		}
 
-		// Record file size
-		if size, err := u.fileSize(episodePath); err != nil {
-			return errors.Wrap(err, "failed to get episode file size")
```
Comment (mxpv) [inline cmd/podsync/updater.go]: We still need this to get file size for existing episodes (the ones, that we didn't download during this update). 

I suggest something like this instead:
```suggestion
// Don't return on error, use estimated file size provided by builders
logger.WithErr(err).Error("failed to get episode file size")
```

Comment (b-abadie) [reply cmd/podsync/updater.go]: You are absolutely right (and you know your project way better than I). I added this last night to have a podcast working before hopping on the subway for my commute. I'll update the PR.

**Comment by b-abadie** [comment]
Hum. Checks fail and I can't fix it right now. Feel free to handle the issue if you want.

**Comment by mxpv** [comment]
@b-abadie sorry, I mistyped my suggestion. Fixed.

**Pull request merged**